libs/designsystem/checkbox/src/checkbox.component.ts

```diff
@@ -28,7 +35,7 @@ export class CheckboxComponent {
 
   @HostBinding('class')
   @Input()
-  size?: 'xs' | 'sm' | 'md';
+  size?: 'xs' | 'sm' | 'md' = 'md';
```

Comment:
Nitty gritty, but I guess this is no longer optional (i.e. can be `undefined`) if it has a default value? 🤔 
```suggestion
  size: 'xs' | 'sm' | 'md' = 'md';
```